app/plugin_install.go

```diff
@@ -16,6 +17,14 @@ import (
 	"github.com/mattermost/mattermost-server/utils"
 )
 
+func (a *App) InstallPluginFromData(data model.PluginEventData) {
+	mlog.Info(fmt.Sprintf("InstallPluginFromData. ID: %v, Path: %v", data.PluginId, data.PluginPath))
```

Comment:
@marianunez these are two apis i've created that will be called on the other clusters to notify them of `install` plugin even